Translations/bg.po

```diff
@@ -3646,1051 +3636,1055 @@ msgstr "Псалтски амулет"
 msgid "Player is on a Hellfire only level"
-msgstr ""
+msgstr "Играчът е на ниво само за Hellfire"
 
-#: Source/loadsave.cpp:1790
+#: Source/loadsave.cpp:1884
 msgid "Invalid game state"
-msgstr ""
+msgstr "Невалидно игрово състояние"
 
-#: Source/mainmenu.cpp:123
+#: Source/menu.cpp:138
 msgid "Unable to display mainmenu"
-msgstr ""
+msgstr "Невъзможно да се покаже главно меню"
 
 #. TRANSLATORS: Monster Block start
 #: Source/monstdat.cpp:19
 msgid "Zombie"
-msgstr ""
+msgstr "Зомби"
 
 #: Source/monstdat.cpp:20
 msgid "Ghoul"
-msgstr ""
+msgstr "Таласъм"
 
 #: Source/monstdat.cpp:21
 msgid "Rotting Carcass"
-msgstr ""
+msgstr "Гниещ Труп"
 
 #: Source/monstdat.cpp:22
 msgid "Black Death"
-msgstr ""
+msgstr "Черна Смърт"
 
 #: Source/monstdat.cpp:23 Source/monstdat.cpp:31
 msgid "Fallen One"
-msgstr ""
+msgstr "Изпаднал"
 
 #: Source/monstdat.cpp:24 Source/monstdat.cpp:32
 msgid "Carver"
-msgstr ""
+msgstr "Дълбач"
 
 #: Source/monstdat.cpp:25 Source/monstdat.cpp:33
 msgid "Devil Kin"
-msgstr ""
+msgstr "Дяволска Челяд"
 
 #: Source/monstdat.cpp:26 Source/monstdat.cpp:34
 msgid "Dark One"
-msgstr ""
+msgstr "Очернен"
 
 #: Source/monstdat.cpp:27 Source/monstdat.cpp:39
 msgid "Skeleton"
-msgstr ""
+msgstr "Скелет"
 
 #: Source/monstdat.cpp:28
 msgid "Corpse Axe"
-msgstr ""
+msgstr "Корпус Брадвен"
 
 #: Source/monstdat.cpp:29 Source/monstdat.cpp:41
 msgid "Burning Dead"
-msgstr ""
+msgstr "Горящ Мъртвец"
 
 #: Source/monstdat.cpp:30 Source/monstdat.cpp:42
 msgid "Horror"
-msgstr ""
+msgstr "Кошмар"
 
 #: Source/monstdat.cpp:35
 msgid "Scavenger"
-msgstr ""
+msgstr "Лешояд"
 
 #: Source/monstdat.cpp:36
 msgid "Plague Eater"
-msgstr ""
+msgstr "Чумояд"
 
 #: Source/monstdat.cpp:37
 msgid "Shadow Beast"
-msgstr ""
+msgstr "Звяр от Сенките"
 
 #: Source/monstdat.cpp:38
 msgid "Bone Gasher"
-msgstr ""
+msgstr "Костоглозгач"
 
 #: Source/monstdat.cpp:40
 msgid "Corpse Bow"
-msgstr ""
+msgstr "Корпус Лъков"
 
 #: Source/monstdat.cpp:43
 msgid "Skeleton Captain"
-msgstr ""
+msgstr "Скелет Капитан"
 
 #: Source/monstdat.cpp:44
 msgid "Corpse Captain"
-msgstr ""
+msgstr "Корпус Капитан"
 
 #: Source/monstdat.cpp:45
 msgid "Burning Dead Captain"
-msgstr ""
+msgstr "Горящ Мъртвец Капитан"
 
 #: Source/monstdat.cpp:46
 msgid "Horror Captain"
-msgstr ""
+msgstr "Кошмар Капитан"
 
 #: Source/monstdat.cpp:47
 msgid "Invisible Lord"
-msgstr ""
+msgstr "Невидим Повелител"
 
-#: Source/monstdat.cpp:48 Source/objects.cpp:92
+#: Source/monstdat.cpp:48 Source/objects.cpp:100
 msgid "Hidden"
-msgstr ""
+msgstr "Прикрит"
 
 #: Source/monstdat.cpp:49
 msgid "Stalker"
-msgstr ""
+msgstr "Следящ от Сенките"
 
 #: Source/monstdat.cpp:50
 msgid "Unseen"
-msgstr ""
+msgstr "Незабелязан"
 
 #: Source/monstdat.cpp:51
 msgid "Illusion Weaver"
-msgstr ""
+msgstr "Предач на Илюзии"
 
 #: Source/monstdat.cpp:52
 msgid "Satyr Lord"
-msgstr ""
+msgstr "Сатир Господар"
 
 #: Source/monstdat.cpp:53 Source/monstdat.cpp:61
 msgid "Flesh Clan"
-msgstr ""
+msgstr "Плътен Клан"
```

Comment:
```suggestion
msgstr "Плътски Клан"
```

Reply:
Drop the 2nd capital letter, else its fine.